src/app.tsx

```diff
@@ -1,24 +1,27 @@
 import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
 import { Box, Text } from "ink";
-import React from "react";
+import { Provider } from "jotai";
+import { store } from "./atoms/store.js";
 import { BoardView } from "./board-view.js";
 
 const queryClient = new QueryClient();
 
 export const App = () => {
   return (
-    <QueryClientProvider client={queryClient}>
-      <Box flexDirection="column">
-        <Text> JIRA TIME</Text>
-        <Box
-          height={"100%"}
-          width={"100%"}
-          borderStyle={"round"}
-          flexDirection="column"
-        >
-          <BoardView />
+    <Provider store={store}>
```

Comment:
Very minor; maybe rename the import to `JotaiProvider` or `StoreProvider` or something, dunno